dask/bag/text.py

```diff
@@ -3,8 +3,9 @@
 from functools import partial
 
 from tlz import concat
+from fsspec.core import open_files
 
-from ..bytes import open_files, read_bytes
+from ..bytes import read_bytes
```

Comment:
I assume that it is preferable to import directly from fsspec at this point to make the code more legible. I can undo these changes though if people disagree.

Reply:
Hmm I don't quite follow why importing from `..bytes` makes things more legible, though I could be missing something. If `read_bytes` is ultimately coming from `fsspec` then my initial thought is that we should import it directly from `fsspec`